updated cover sheets for TN-9.7.3 and S-9.7.4.1 and Summary-of-Changes

kiwi64ajs opened the pull request:
This PR
- updated cover sheets for TN-9.7.3-MessageNetwork and S-9.7.4.1-ConfigurationDescriptionInformation with changes
- updated the Summary-of-Changes-to-LCC-Documents-2021-04-25

balazsracz reviewed (approved):
Looks fine to me. The only wish I have is for us to add some designation of where a given change happened. For example "Section a.b.c: corrected xyz".

kiwi64ajs commented:
Ok, I’ve added Section and Line numbers etc to better define the location of the changes and committed those changes to PR #42 here:

	https://github.com/openlcb/documents/pull/42/commits/cd6ebf84cb59da1029f9007ff60d2379516157a9

Please check ASAP as I want to send these tomorrow AM.

Alex 

> On 5/06/2021, at 7:24 PM, Balazs Racz ***@***.***> wrote:
> 
> 
> @balazsracz approved this pull request.
> 
> Looks fine to me. The only wish I have is for us to add some designation of where a given change happened. For example "Section a.b.c: corrected xyz".
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub <https://github.com/openlcb/documents/pull/42#pullrequestreview-676758622>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AB5Y53MIWTMEN33VT5CJOXLTRHGMFANCNFSM45LRQQHA>.
>

balazsracz reviewed (approved):
thanks for updating, looks good to me!

dpharris commented:
Agree, looks good.  Thanks Alex!

David


On Wed., Jun. 9, 2021, 05:38 Alex Shepherd, ***@***.***>
wrote:

> Ok, I’ve added Section and Line numbers etc to better define the location
> of the changes and committed those changes to PR #42 here:
>
>
> https://github.com/openlcb/documents/pull/42/commits/cd6ebf84cb59da1029f9007ff60d2379516157a9
>
> Please check ASAP as I want to send these tomorrow AM.
>
> Alex
>
> > On 5/06/2021, at 7:24 PM, Balazs Racz ***@***.***> wrote:
> >
> >
> > @balazsracz approved this pull request.
> >
> > Looks fine to me. The only wish I have is for us to add some designation
> of where a given change happened. For example "Section a.b.c: corrected
> xyz".
> >
> > —
> > You are receiving this because you were assigned.
> > Reply to this email directly, view it on GitHub <
> https://github.com/openlcb/documents/pull/42#pullrequestreview-676758622>,
> or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/AB5Y53MIWTMEN33VT5CJOXLTRHGMFANCNFSM45LRQQHA
> >.
> >
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/openlcb/documents/pull/42#issuecomment-857658545>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEDQSVBVDHWAGAJYEEFZ5LTR5OEVANCNFSM45LRQQHA>
> .
>
